pkgs/dart_model/benchmark/json_buffer.dart

```diff
@@ -3,20 +3,88 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'dart:typed_data';
+
 import 'package:benchmark_harness/benchmark_harness.dart';
+import 'package:collection/collection.dart';
 import 'package:dart_model/src/json_buffer.dart';
 
+Uint8List? serialized0;
 Uint8List? serialized;
+Uint8List? serialized2;
 JsonBuffer? deserialized;
-const mapSize = 1000000;
+const mapSize = 10000;
 final mapKeys = List.generate(mapSize, (i) => i.toString());
 
 void main() {
+  JsonBufferSerializeBenchmark0().report();
```

Comment:
Fwiw calling each benchmark 3 times shouldn't be necessary... it does its own warmup. It is possible the warmup isn't running enough times or something? We can override how warmup works if we need which might be better?

Reply:
I added it to get an idea of the variance, then noticed that indeed the first run of the first benchmark was slower. Looking closer the runs do not seem to be particularly stable, sometimes there is a first benchmark difference, sometimes not, and there are quite large differences between runs ... I should really be running AOT compiled anyway, I guess, will switch to doing that.